partitura/io/exportaudio.py

```diff
@@ -39,15 +41,29 @@ def save_wav(
         the audio signal as an array (see `audio_signal` below).
     samplerate: int
         The sample rate of the audio file in Hz. The default is 44100Hz.
-    envelope_fun: {"linear", "exp" }
+    envelope_fun: {"linear", "exp" } or callable
         The type of envelop to apply to the individual sine waves.
-    tuning: {"equal_temperament", "natural"}
+        If "linear" or "exp", the methods `lin_in_lin_out` and `exp_in_exp_out`
+        in `partitura.utils.synth` will be used. Otherwise this argument should
+        be a callable. See `lin_in_lin_out` for more details.
+    tuning: {"equal_temperament", "natural"} or callable
+        Method for tuning. If "equal temperament" will use equally spaced
+        semitones, while "natural" will use natural ratios within an octave.
+        Otherwise it uses a callable. See `midi_pitch_to_natural_frequency`
+        and `midi_pitch_to_frequency` for more info on the "equal_temperament"
+        and "natural" tuning.
```

Comment:
natural temperament is based on the reference pitch of A, this choice is not clearly documented and gives wrong results for many pieces where the base note should be different. I see two ways of fixing this, 1) using the `midi_pitch_to_tempered_frequency` function instead or 2) clearly documenting that this renders a version based on  reference pitch A with interval ratios given in `NATURAL_INTERVAL_RATIOS`

Reply:
I added a symmetric 5-limit temperament table `FIVE_LIMIT_INTERVAL_RATIOS`